history-of-quakers/modernized/12-chapter-10.adoc

```diff
@@ -373,6 +418,10 @@ and in England it being resolved upon to call him back, and to restore him,
 he gave forth the following declaration at Breda;
 thereby to persuade those that were yet backward to acknowledge him.
 
+[.embedded-content-document.letter]
+--
+
+[.salutation]
```

Comment:
This letter is a little confusing. It seems it from Charles Stuart but looks like it's addressed to Charles R, but it was actually a declaration to a lot of people. So idk if there's something I'm missing, or if it's just a weird old style of how they used to write stuff? 🤷

Reply:
yes, confusing indeed. I googled it a bit, and I'm pretty sure that this is Charles II's _Declaration at Breda_ and his signature is `Charles R.`.  So it's written by Charles "Stuart".  To clarify this, let's do two things:

1. change it from `letter` to `legal`
2. move the `Charles R.` down to the bottom, before the [sscc] and make it a signature